### DISCUSSION TITLE
changing overlay directory

### DISCUSSION THREAD

**Discussion by celesteking** [oamg/leapp-repository#1264] [Q&A] [answered]
How the heck do I change overlay `/var/lib/leapp` directory? Is it hard-coded? I can't seem to find that anywhere in the code.

**Comment by pirat89** [comment, marked as answer]
Hi @celesteking, well, it depends what you mean by overlay directory, as for us overlay directory (directories actually) is under `/var/lib/leapp/scratch`). Well, you could use in theory `LEAPP_CONTAINER_ROOT` envar to set a different dir, but not sure whether it will do exactly what you expect, as it's possible you have something else in your mind than me.

For more info, look at the `targetuserspacecreator` actor and the `TargetUserSpaceInfo` model.
